myokit/_sim/cvodessim.py

```diff
@@ -158,13 +162,19 @@ def __init__(self, model, protocol=None, sensitivities=None, path=None):
         del model
 
         # Set protocol
-        self._protocol = None
-        self._fixed_form_protocol = None
-        self.set_protocol(protocol)
-        del protocol
+        if isinstance(protocols, myokit.Protocol):
+            protocols = {'pace': protocols}
+        if protocols is None:
+            protocols = {'pace': myokit.Protocol()}
```

Comment:
mainly for backwards compatability I think. Previously you were always binding the `pace` variable, so we still want to do that?

Reply:
ok, I'll give that a go

Reply:
nope, this has some unfortunate effects. Previously the `pace` binding was always kept, but if you don't bind it here, then it is removed which would break users code subsequently. Think we have to keep as is until #320 is resolved